tensorflow/core/kernels/cwise_ops_test.cc

```diff
@@ -49,21 +41,23 @@ static int RowsAndColsArg(int r, int c) { return r * kRows + c; }
 static int RowsFromArg(int arg) { return (arg / kRows); }
 static int ColsFromArg(int arg) { return (arg % kRows); }
 
-#define BM_UNARY(DEVICE, FUNC, TYPE)                              \
-  static void BM_##DEVICE##_##FUNC_##TYPE(int iters, int num) {   \
-    const int64 tot = static_cast<int64>(iters) * num;            \
-    testing::ItemsProcessed(tot);                                 \
-    testing::BytesProcessed(tot * sizeof(float));                 \
-    test::Benchmark(#DEVICE, Unary(#FUNC, num, TYPE)).Run(iters); \
-  }                                                               \
+#define BM_UNARY(DEVICE, FUNC, T, TYPE)                              \
+  static void BM_##DEVICE##_##FUNC_##TYPE(int iters, int num) {      \
+    const int64 tot = static_cast<int64>(iters) * num;               \
+    testing::ItemsProcessed(tot);                                    \
+    testing::BytesProcessed(tot * sizeof(float));                    \
```

Comment:
indeed! Funnly enough the test passed which is kind of confusing... I am actually having issues testing the gpu part on my mac (it crashes while loading the curand dynamic library) and on ubuntu it didnt fail so I don't know how to see a failure...

What bazel command do you use to test?